common/JackRequest.h

```diff
@@ -90,7 +90,7 @@ struct JackRequest
         kGetUUIDByClient = 37,
         kClientHasSessionCallback = 38,
         kComputeTotalLatencies = 39,
-        kPropertyChangeNotify = 40
+        kPropertyChangeNotify = 40,
```

Comment:
this comma is really not needed for anything.
it actually breaks compilation if you include the header in something being compiled in strict/pedantic mode.

Reply:
aha wasnt aware of that, the main reason for it is if you add a new enum or member, you modify only one line and resulting diff is therefore smaller and highlights only important change

Reply:
yeah, that is usually a good reason. but "strictly" speaking, it breaks pedantic language checks.